tests/testthat/test-scale.R

```diff
@@ -115,6 +117,13 @@ test_that("warns on zv",{
   expect_snapshot(prep(rec1))
 })
 
+test_that("warns when NaN is returned",{
+  rec1 <- rec %>%
+    step_log(sulfur) %>%
+    step_scale(sulfur)
+  expect_warning(prep(rec1))
```

Comment:
Please switch to use `expect_snapshot()` so that we can verify that it is the correct warning that is omitted 😄

Reply:
Done!